scripts/install-cni.sh

```diff
@@ -1,27 +1,57 @@
-#!/bin/sh
-
+#! /bin/bash
 set -e -x;
 
-if [ -w "/host/opt/cni/bin/" ]; then
-    cp -f /install/bin/* /host/opt/cni/bin/;
-    echo "Wrote CNI binaries to /host/opt/cni/bin/";
+CONF_FILE_PATH="/install/config"
+BIN_FILE_PATH="/install/bin"
+CONF_FILE_PATH_HOST="/host/etc/cni/net.d/"
```

Comment:
Im not sure about this, but CONF_FILE_PATH_SRC and CONF_FILE_PATH_DEST will be more accurate ? @jkraj

Reply:
done